src/main/resources/schemas.yml

```diff
@@ -54,4 +54,13 @@ components:
       type: object
       properties:
         message:
-          type: string
\ No newline at end of file
+          type: string
+    Post:
+      type: object
+      properties:
```

Comment:
As one of the comments mentioned, change imageId to string and add id type of string, which will be mapped